### PR TITLE
User-shared networks and network permissions

### PR DESCRIPTION
**Pull request by weizhouapache**
This documents https://github.com/apache/cloudstack/pull/5769

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
links for some changes

http://qa.cloudstack.cloud/docs/WIP-PROOFING/pr/258/adminguide/networking_and_traffic.html#configuring-a-shared-guest-network
http://qa.cloudstack.cloud/docs/WIP-PROOFING/pr/258/adminguide/networking_and_traffic.html#adding-a-network-permission

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

File: source/adminguide/networking.rst [hunk 1]
```diff
@@ -71,7 +71,9 @@ different accounts. Network Isolation on shared networks is accomplished
 by using techniques such as security groups, which is supported only in
 Basic zones or Advanced Zones with Security Groups.
 
--  Shared Networks are created by the administrator
+-  Shared Networks are created by the the end users, however network offerings
```
Comment (nvazquez) [inline source/adminguide/networking.rst:74]:
```suggestion
-  Shared Networks are created by the the end users or the administrator. Network offerings
```

File: source/adminguide/networking/advanced_zone_config.rst [hunk 2]
```diff
@@ -123,13 +128,17 @@ Configuring a Shared Guest Network
...
-      -  **All**: The guest network is available for all the domains,
-         account, projects within the selected zone.
+      -  **All**: (Administrators only) The guest network is available
+         for all the domains, account, projects within the selected zone.
 
    -  **Network Offering**: If the administrator has configured multiple
       network offerings, select the one you want to use for this
       network.
 
+   -  **Associated Network**: The L2 or Isolated network this network is
```
Comment (nvazquez) [inline source/adminguide/networking/advanced_zone_config.rst:138]: Should this be here as well or only on Add new VPC gateway?

File: source/adminguide/networking/network_permissions.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+   under the License.
+
+
+Guest Network Permissions
+-----------------------------
+
+From Apache CloudStack 4.17.0.0, guest networks can be shared to other
+accounts in the same domain by managing network permissions.
+
+The following networks can be shared:
```
Comment (nvazquez) [inline source/adminguide/networking/network_permissions.rst:23]: I think a new line at the end of this line would display the items below as a list, currently displayed in the same line:
Screen Shot 2022-04-20 at 00 13 34

File: source/adminguide/networking/advanced_zone_config.rst (hunk 2 above)
Comment (weizhouapache) [reply source/adminguide/networking/advanced_zone_config.rst:138]: @nvazquez 
associated network is available for both user-shared network and user-vpc private gateway.

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

File: source/adminguide/networking.rst (hunk 1 above)
Comment (weizhouapache) [reply source/adminguide/networking.rst]: thanks @nvazquez 
committed

**Comment by weizhouapache** [comment]
@blueorangutan docbuild

File: source/adminguide/networking/network_permissions.rst (hunk 3 above)
Comment (weizhouapache) [reply source/adminguide/networking/network_permissions.rst:23]: @nvazquez solved. thanks. now it looks like
image

**Comment by nvazquez** [comment]
Thanks @weizhouapache, LGTM

**Review by nvazquez — APPROVED** [review]

**Pull request merged**